tests/datafile/shuffletest.tim

```diff
@@ -0,0 +1,123 @@
+FORMAT 1
+#J0523-7125_PSMC.rf.f4T 1382.000000 55415.829511131335920 4407.496  pks 
```

Comment:
Is there any reason this can't be used as an example file with multiple telescopes for other test code? The name suggests a more specific nature than we necessarily need. If it's really only good for the one use, maybe trim it to three or so TOAs from each telescope and put it in a triple-quoted string?

Reply:
It's from an unpublished project, and I didn't want to include the par file (which might have been helpful for other tests).  Ultimately for this test the TOAs don't really matter so I could just make up numbers and put it as a string - I was just exploring the testing possibilities.

Reply:
I changed it to fake data.